src/ctap/storage.rs

```diff
@@ -622,6 +625,9 @@ impl PersistentStore {
 
     /// Enables alwaysUv, when disabled, and vice versa.
     pub fn toggle_always_uv(&mut self) -> Result<(), Ctap2StatusCode> {
+        if ENFORCE_ALWAYS_UV {
+            return Ok(());
```

Comment:
Would it make sense to directly return `Ctap2StatusCode::CTAP2_ERR_OPERATION_DENIED` here? Alternatively return `INTERNAL_ERROR`.

Reply:
I wasn't sure if I want that logic to live inside `storage.rs`, but it's more strict, so can't hurt. :) Changed.